enhancements/update/clusterversion-osus-signatures.md

```diff
@@ -0,0 +1,138 @@
+
+
+## Motivation
+
+Today, in the disconnected environments, a user has to clone the signatures config map as well as the release images to update the clusters.
+With [serve image signatures for disconnected environments], OSUS will be able to serve the cloned image signatures locally
+
+There is no existing way in CVO to fetch the signatures through a custom URI. This enhancement defines an API change in the CVO
+to accept a custom update-service URI which can be used to verify release signatures. 
+
+
+### User Stories
+
+* As a cluster administrator, I want the cluster's CVO to add a new signature store URI.
+
+* As a cluster administrator, I want the cluster's CVO to replace the existing signature store URIs with a user provided URI
+
+### Goals
+
+* To verify the release image signature in disconnected environments using OSUS signatures URI.
+* To replace existing signature stores with custom user-provided stores.
+* Implementation will require an API change.
+
+### Non-Goals
+
+* We will not verify the authenticity of the verification signature source.
+
+## Proposal
+
+### Workflow Description
+
+1. Cluster admin will have to add the verification URI to CVO
+2. CVO will reach the newly added URI to verify the release images.
+
+### API Extensions
+
+CVO will grow a new field which will be populated with the image verification URI.
+
+### Risks and Mitigations
+
+* As we are not verifying the authenticity of the signature source, a malicious threat actor with access to cluster admin resource
+  could be able to add malicious images and get them verified locally with malicious signatures.
+  * This can still be possible today by using `--force` while upgrading the cluster through cli.
+
+### Drawbacks
+
+* The cluster admin will have to have their own due diligence to make sure that the release images and verification signatures are
+  authentic.
+
+## Design Details
+
+### Test Plan
+
+* CVO unit tests will be expanded and/or created to test the new logic.
+
+### Graduation Criteria
+
+GA. When it works, we ship it.
+
+#### Dev Preview -> Tech Preview
+
+N/A. This is not expected to be released as Dev Preview.
+
+#### Tech Preview -> GA
+
+N/A. This is not expected to be released as Tech Preview.
+
+#### Removing a deprecated feature
+
+N/A.
+
+### Upgrade / Downgrade Strategy
+
+No special consideration.
+
+### Version Skew Strategy
+
+No special consideration.
+
+### Operational Aspects of API Extensions
+
+....[WIP]
+
+#### Failure Modes
+
+No special consideration.
+
+#### Support Procedures
+
+No special consideration.
+
+## Implementation History
+
+
+## Alternatives
+
+### Automatically add OSUS signature verification URI to CVO
+
+We can use the OSUS instance configured locally to replace the signature verification URI, or add the local
+image verification URI to existing stores which the CVO can then use to verify the release image. This will work automatically, 
```

Comment:
I am not sure what you want to suggest here. Can you add more context?

Reply:
changed the wording to be more clear. 
alternative is basically, CVO will directly get the signature verification URL from cluster hosted OSUS instance. But this won't work on clusters that depend in OSUS that's hosted on a separate cluster.